### PR TITLE
Remove sepolia/race

### PR DESCRIPTION
**Pull request by geoknee**

### CODE REVIEW

File: superchain/extra/addresses/addresses.json [hunk 1]
```diff
@@ -18,7 +18,10 @@
     "PreimageOracle": "0xD326E10B8186e90F4E2adc5c13a2d0C137ee8b34",
     "MIPS": "0x0f8EdFbDdD3c0256A80AD8C0F2560B1807873C9c",
     "Guardian": "0x09f7150D8c019BeF34450d6920f6B3608ceFdAf2",
-    "Challenger": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A"
+    "Challenger": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
```
Comment (bitwiseguy) [inline superchain/extra/addresses/addresses.json:21]: Are the `Challenger`, `Proposer`, `UnsafeBlockSigner` and `BatchSubmitter` address changes meant to be part of this PR? Seems like they are lingering around but not related to this PR

**Review by bitwiseguy — CHANGES_REQUESTED** [review]
I think there are some address changes that were unintentional included in this PR

File: superchain/extra/addresses/addresses.json (hunk 1 above)
Comment (geoknee) [reply superchain/extra/addresses/addresses.json:21]: Correct, they happened when I ran `pnpm codegen` to update this file. I filed an issue about this https://github.com/ethereum-optimism/superchain-registry/issues/345 . 

For now I will just revert the changes to this file and tackle the above issue separately.

Comment (geoknee) [reply superchain/extra/addresses/addresses.json]: [d601035](https://github.com/ethereum-optimism/superchain-registry/pull/342/commits/d60103538ffd761df39a03693a50091312cf7b23)

**Review by bitwiseguy — APPROVED** [review]
LGTM!

**Pull request merged**